app-config.yaml

```diff
@@ -224,6 +224,17 @@ catalog:
     #   providerId:
     #     organization: "${GITHUB_ORG}"
 
+    microsoftGraphOrg:
+      default:
+        tenantId: ${AZURE_TENANT_ID}
+        user:
```

Comment:
I would remove the `user` and `group` sections.  Those are just samples

Reply:
done